### PR TITLE
DropDownList: Add aria-label to the List

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js [hunk 1]
```diff
@@ -1710,6 +1710,21 @@ QUnit.module('aria accessibility', moduleConfig, () => {
...
+        });
+
+        assert.strictEqual($(`.${LIST_CLASS}`).attr('aria-label'), 'List', 'aria-label is set correctly');
+    });
+
+    QUnit.test('list aria-label should be equal "No data to display", if it has not items', function(assert) {
+        $('#dropDownList').dxDropDownList({ opened: true });
+
+        assert.strictEqual($(`.${LIST_CLASS}`).attr('aria-label'), 'No data to display', 'aria-label is set correctly');
+    });
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js:1726]: let's test `items/dataSource` runtime change

Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js:1726]: Was added in [1a94a48e49f47a4ff98b06c38c7f2f130d0032ca].

File: js/ui/drop_down_editor/ui.drop_down_list.js [hunk 2]
```diff
@@ -465,6 +465,14 @@ const DropDownList = DropDownEditor.inherit({
         this._list._getAriaTarget = this._getAriaTarget.bind(this);
     },
 
+    _setAriaLabelToList() {
```
Comment (ksercs) [inline js/ui/drop_down_editor/ui.drop_down_list.js]:
```suggestion
    _setListAriaLabel() {
```

File: js/ui/drop_down_editor/ui.drop_down_list.js [hunk 3]
```diff
@@ -873,10 +881,12 @@ const DropDownList = DropDownEditor.inherit({
             case 'items':
                 if(!this.option('dataSource')) {
                     this._processDataSourceChanging();
+                    this._setAriaLabelToList();
```
Comment (ksercs) [inline js/ui/drop_down_editor/ui.drop_down_list.js]: let's move it to `_processDataSourceChanging` to reduce copypaste

File: testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js [hunk 4]
```diff
@@ -1710,6 +1710,30 @@ QUnit.module('aria accessibility', moduleConfig, () => {
         assert.deepEqual(list._getAriaTarget(), dropDownList._getAriaTarget());
         assert.strictEqual($input.attr('role'), 'combobox', 'input.role');
     });
+
+    QUnit.test('list aria-label should be set correctly if items changes in runtime', function(assert) {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js]: it tests not only a runtime change
```suggestion
    QUnit.test('list aria-label should be set correctly', function(assert) {
```

File: testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js [hunk 5]
```diff
@@ -1710,6 +1710,30 @@ QUnit.module('aria accessibility', moduleConfig, () => {
...
+        assert.strictEqual($(`.${LIST_CLASS}`).attr('aria-label'), 'No data to display');
+
+        instance.option('items', [1, 2, 3]);
+        assert.strictEqual($(`.${LIST_CLASS}`).attr('aria-label'), 'List');
+
+        instance.option('items', []);
+        assert.strictEqual($(`.${LIST_CLASS}`).attr('aria-label'), 'No data to display');
+    });
+
+    QUnit.test('list aria-label should be set correctly if dataSource changes in runtime', function(assert) {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js]: we can reduce copypaste if we use `['items', 'dataSource].forEach`

File: js/ui/drop_down_editor/ui.drop_down_list.js (hunk 2 above)
Comment (marker-dao) [reply js/ui/drop_down_editor/ui.drop_down_list.js]: Very good variable name. I will use it. Thanks. Look at [d55def719df90757723ca998b13c81a9f9a52e0b].

File: js/ui/drop_down_editor/ui.drop_down_list.js (hunk 3 above)
Comment (marker-dao) [reply js/ui/drop_down_editor/ui.drop_down_list.js]: Good idea! Was fixed in [d55def719df90757723ca998b13c81a9f9a52e0b].

File: testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js (hunk 4 above)
Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js]: Thanks for your attention! Was fixed in [d55def719df90757723ca998b13c81a9f9a52e0b].

File: testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js (hunk 5 above)
Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js]: Yes! Was fixed in [d55def719df90757723ca998b13c81a9f9a52e0b].

**Review by Zedwag — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**